sessionize/lib/src/commonMain/kotlin/co/touchlab/sessionize/platform/Tests.kt

```diff
@@ -0,0 +1,26 @@
+package co.touchlab.sessionize.platform
+
+import co.touchlab.stately.annotation.Throws
+import kotlinx.serialization.ImplicitReflectionSerializer
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.JsonArray
+import kotlinx.serialization.parseList
+import kotlin.test.assertNotEquals
+import kotlin.test.assertTrue
+
+
+@UseExperimental(ImplicitReflectionSerializer::class)
+@Throws
+fun testSponsorSeedFile() {
```

Comment:
@samhill303 Suggested that we move this to commonTest but we don't yet know how to package it up for iOS.